### PR TITLE
add angularjs-color-picker v1.0.3

### LINKED ISSUE #7565: [Request] Add angular-color-picker
**Library name:** angular-color-picker
**Git repository url:** https://github.com/ruhley/angular-color-picker
**npm package url(optional):** https://www.npmjs.com/package/angularjs-color-picker
**License(s):** MIT
**Official homepage:** http://ruhley.github.io/angular-color-picker/
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by dannyxx001**
@pvnr0082t please help me review this pull request, thanks.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/ruhley/angular-color-picker
 * Official website (optional, not the repository): http://ruhley.github.io/angular-color-picker/
 * NPM package url (optional): https://www.npmjs.com/package/angularjs-color-picker
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 56
   - Count of watchers: 3
   - Count of forks: 39
 * NPM download stats (optional):
   - Downloads in the last day: 61
   - Downloads in the last week: 265
   - Downloads in the last month: 641

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

with autoupdate config

close #7565, cc @ruhley

### CODE REVIEW

File: ajax/libs/angularjs-color-picker/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "angular": "^1.4.0",
+    "tinycolor2": "^1.3.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/ruhley/angular-color-picker.git"
+  },
+  "authors": [
+    "ruhley <timothy.ruhle@gmail.com>"
+  ],
```
Comment (pvnr0082t) [inline ajax/libs/angularjs-color-picker/package.json]: Let's use ```"author": "ruhley <timothy.ruhle@gmail.com>"``` here.

File: ajax/libs/angularjs-color-picker/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  "homepage": "https://github.com/ruhley/angular-color-picker#readme",
```
Comment (PeterDaveHello) [inline ajax/libs/angularjs-color-picker/package.json]: Homepage to git repo should be also removed.

**Comment by dannyxx001** [comment]
OK, I have updated it. Thank you.

**Comment by PeterDaveHello** [comment]
@dannyxx001 don't forget to tag your reviewer to review again.

**Pull request merged**